cmd/frontend/graphqlbackend/campaigns.go

```diff
@@ -180,6 +180,13 @@ type ChangesetCountsArgs struct {
 	To   *DateTime
 }
 
+type ListChangesetsArgs struct {
+	First       *int32
+	State       *string
+	ReviewState *string
+	CheckState  *string
```

Comment:
Can we use our types for this? Since we return them in the `ExternalChangesetResolver` methods it would be nice if we can use them consistently here.

```suggestion
	State       *campaigns.ChangesetState
	ReviewState *campaigns.ChangesetReviewState
	CheckState  *campaigns.ChangesetCheckState
```